modules/sql-engine/src/test/resources/mapping/test_partition_pruning.test

```diff
@@ -0,0 +1,53 @@
+# Pruning from a simple scan
+N1
+SELECT * FROM t1_n1n2n3 WHERE id = 0
```

Comment:
let's add one more test case to check pruning for collocated join

Reply:
Fixed.